monai/networks/trt_compiler.py

```diff
@@ -233,13 +321,15 @@ def __init__(
         method="onnx",
         input_names=None,
         output_names=None,
+        output_lists=None,
```

Comment:
Could you please also add a simple test case to the unit test to show case how to use this parameter?

Thanks,  
Bin

Reply:
Done.